cards.css

```diff
@@ -1,58 +1,135 @@
-dl {
-    display: grid;
-    grid-template-columns: max-content 200px;
+body {
+    min-height: 100vh;
+    margin: 0;
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+    align-items: center;
+    font-family: Inter, sans-serif;
+    font-feature-settings: 'liga' 1, 'calt' 1; /* fix for Chrome */
+  }
+
+.card-mobile {  
+    border: 1px solid #000;
+    border-radius: .5em;
+    display: inline-flex;
+    padding: 20px;
+    font: 1em Helvetica Neue, Helvetica, Arial, sans-serif;
+    box-shadow: 3px 3px 5px #00000025;
+  
   }
   
-  dt {
-    grid-column-start: 1;
-    padding-bottom: 20px;
-    margin-right: -30px;
+  * {box-sizing: border-box;}
+  
   
+  .child {
+      border-width: 0 2px 2px 2px;
   }
   
-  dd {
-    grid-column-start: 2;
+  .parent {
+      width: 308px; /* or any other width */
+      height: auto; /* or any other fixed height */
   }
   
-  span {
-    background-color: #777;
-    border-radius: 4px;
-    padding-left: 10px;
-    padding-right: 10px;
-    padding-top: 2px;
-    padding-bottom: 2px;
+  .header-blue p {
+    margin: 14px;
   }
   
-  * {box-sizing: border-box;}
+  .header-blue {
+    display: table;
+    margin: 0 auto;
+    background: linear-gradient(#315394,#0A0F22);
+    width: 286px;
+    font-size: 25px;
+    text-align: center;
+    color: white;
+    border-radius: 6px;
   
+  }
```

Comment:
Just a hint for not having to repeat...
(but I know you are rushing to get this in...)
Your class names "header-blue", etc. could be split into
```css
header {
    display: table;
    ...
    border-radius: 6px;
}
```
without the `background: linear-gradient()`,
then
```css
blue {
    background: linear-gradient()
}
green {
    background: linear-gradient()
}
```
for each color.  Then you can also just have a single
```css
header p {
    margin: 14px;
}
```